docs/dev/libvirt-howto.md

```diff
@@ -172,6 +172,10 @@ This step is optional, but useful for being able to resolve cluster-internal hos
 ```sh
 GOBIN=~/.terraform.d/plugins go get -u github.com/dmacvicar/terraform-provider-libvirt
 ```
+3. Make sure terraform provider in your executable path:
```

Comment:
This isn't needed since Terraform has its own internal search path.

Reply:
@crawford no, if you don't put your terraform provider in the path the `openshift-install cluster` doesn't found it and complain about same.

Reply:
That's not true. I don't have `terraform-provider-libvirt` in my path and it works fine. It sounds like there is a problem with your installation.

Reply:
> So our GOBIN should be sufficient (with the possible exception of Windows). If you have your terraform-provider-libvirt in ~/.terraform.d/plugins and terraform init is not finding it, maybe file a Terraform issue?

@wking If `openshift-install cluster` does `terraform init` in the background then yes I will gather some more info and file it from terraform side.